packages/server/src/schema/uuid/abstract-entity/resolvers.ts

```diff
@@ -588,6 +712,65 @@ async function addRevision({
   }
 }
 
+async function getEntity<S extends Model<'AbstractEntity'> | null>(
```

Comment:
```suggestion
async function getEntity<S extends Model<'AbstractEntity'>>(
```
Result cannot be `null` due to implementation